actions/terra-k8s-linter/entrypoint.sh

```diff
@@ -36,4 +36,5 @@ else
   fi
 fi
 
-conftest test "${MANIFESTS_DIR}" -p "${policy_dirs}"
+# Run policies against manifests in directory except those associated with daterepo
```

Comment:
just for clarity, is there a reason you can add for the reason we're exempting datarepo? e.g. seems like something we'd one day want to unexempt when the time is right.

Reply:
There's no important reason. It doesn't affect the way the program runs. It's excluded because the dataRepo has so many linting errors that we didn't want to deal with now. If someone deleted the exemption they would be made aware of the linting errors from datarepo. However, for the sake of rolling the linter out, we're excluding them.

Reply:
FWIW I think datarepo things might eventually be consolidated into our repos as we take over their infra